python/mxnet/optimizer.py

```diff
@@ -157,7 +157,7 @@ def set_wd_mult(self, args_wd_mult):
         """
         self.wd_mult = {}
         for n in self.idx2name.values():
-            if not n.endswith('_weight'):
+            if not (n.endswith('_weight') or n.endswith('_gamma')):
```

Comment:
I think decay for gamma is a necessary regularization, you can get better generalization.